microservices-connector/config/samples/ChatQnA/use_cases.md

```diff
@@ -0,0 +1,222 @@
+
+The ChatQnA application is defined as a Custom Resource (CR) file that the above GMC operator acts upon. It first checks if the microservices listed in the CR yaml file are running, if not starts them and then proceeds to connect them. When the ChatQnA RAG pipeline is ready, the service endpoint details are returned, letting you use the application. Should you use "kubectl get pods" commands you will see all the component microservices, in particular `embedding`, `retriever`, `rerank`, and `llm`.
+
+## Using prebuilt images
+
+The ChatQnA uses the below prebuilt images if you choose a Xeon deployment
+
+- embedding: opea/embedding-tei:latest
+- retriever: opea/retriever-redis:latest
+- reranking: opea/reranking-tei:latest
+- llm: opea/llm-tgi:latest
+- dataprep-redis: opea/dataprep-redis:latest
+- tei_xeon_service: ghcr.io/huggingface/text-embeddings-inference:cpu-1.5
+- tei_embedding_service: ghcr.io/huggingface/text-embeddings-inference:cpu-1.5
+- tgi-service: ghcr.io/huggingface/text-generation-inference:sha-e4201f4-intel-cpu
+- redis-vector-db: redis/redis-stack:7.2.0-v9
+
+Should you desire to use the Gaudi accelerator, two alternate images are used for the embedding and llm services.
+For Gaudi:
+
+- tei-embedding-service: opea/tei-gaudi:latest
+- tgi-service: ghcr.io/huggingface/tgi-gaudi:1.2.1
+
+> [NOTE]  
+> Please refer to [Xeon README](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/xeon/README.md) or [Gaudi README](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/README.md) to build the OPEA images. These too will be available on Docker Hub soon to simplify use.
+
+## Deploy ChatQnA pipeline
+
+There are 3 use cases for ChatQnA example:
+
+- General ChatQnA with preset RAG data
+- ChatQnA with data preparation which supports that the user can upload RAG data online via dataprep microservice
+- ChatQnA supports multiple LLM models which can be switched in runtime
+
+### General ChatQnA with preset RAG data
+
+This involves deploying the ChatQnA custom resource. You can use `chatQnA_xeon.yaml` or if you have a Gaudi cluster, you could use `chatQnA_gaudi.yaml`.
+
+1. Create namespace and deploy application
+
+   ```sh
+   kubectl create ns chatqa
+   kubectl apply -f $(pwd)/chatQnA_xeon.yaml
+   ```
+
+2. GMC will reconcile the ChatQnA custom resource and get all related components/services ready. Check if the service up.
+
+   ```sh
+   kubectl get service -n chatqa
+   ```
+
+3. Retrieve the application access URL
+
+   ```sh
+   kubectl get gmconnectors.gmc.opea.io -n chatqa
+   NAME     URL                                                      READY     AGE
+   chatqa   http://router-service.chatqa.svc.cluster.local:8080      8/0/8     3m
```

Comment:
can you confirm it's the latest controller? the latest controller should set the status to 9/0/9 as expected, including the the router's status

Reply:
Hi @KfreeZ, this use case does not support the `dataprep` microservice.  `9/0/9` should be in `ChatQnA with data preparation`

Reply:
Okay, got it, the number is increased